bin/centreondb-smooth-backup.sh

```diff
@@ -41,20 +41,19 @@ BACKUP_DIR="/var/backup"
 SAVE_LAST_DIR="/var/lib/centreon-backup"
 SAVE_LAST_FILE="backup.last"
 DO_ARCHIVE=1
-DB_ROOT_USER="root"
-DB_ROOT_PASSWORD=""
-INIT_SCRIPT="" # try to find it
+DB_ROOT_USER="@MYSQL_REPL_USER@"
+DB_ROOT_PASSWORD="@MYSQL_REPL_PASSWD@"
 PARTITION_NAME="centreon_storage/data_bin centreon_storage/logs"
-PACEMAKER_ON="0"
-PACEMAKER_RSC_MYSQL="ms_MySQL"
+PACEMAKER_ON="1"
+PACEMAKER_RSC_MYSQL="ms_mysql"
 MYSQL_CNF="/etc/my.cnf.d/server.cnf"
-READONLY_CHECK=0
+READONLY_CHECK=1
 
 ###
 # Check MySQL launch
 ###
 process=$(ps -o args --no-headers -C mysqld)
-started=0
+started=1
```

Comment:
since started is set to 1 at the beginning of the script, you can remove all the conditions that checks if it is set to 1 because you never put it back to 0 anyway. (that's just for the sake of the simplicity of the script)